e2e/terraform/shared/scripts/setup.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+set -e
+
+# Disable interactive apt prompts
+export DEBIAN_FRONTEND=noninteractive
+
+cd /ops
+
+CONFIGDIR=/ops/shared/config
+
+CONSULVERSION=1.3.1
+CONSULDOWNLOAD=https://releases.hashicorp.com/consul/${CONSULVERSION}/consul_${CONSULVERSION}_linux_amd64.zip
+CONSULCONFIGDIR=/etc/consul.d
+CONSULDIR=/opt/consul
+
+VAULTVERSION=0.11.4
+VAULTDOWNLOAD=https://releases.hashicorp.com/vault/${VAULTVERSION}/vault_${VAULTVERSION}_linux_amd64.zip
+VAULTCONFIGDIR=/etc/vault.d
+VAULTDIR=/opt/vault
+
+NOMADVERSION=0.8.6
```

Comment:
Installation related variables should be removed here, we likely need to leave the configuration directory setup.